src/pages/HomePage.tsx

```diff
@@ -57,7 +60,7 @@ const HomePage: FC<IProps & RouteComponentProps<IRouteProps>> = ({ match }) => {
 
   return (
     <Box display="flex" flexGrow={1} className={classes.mainBox}>
-      <Box flexGrow={1} width={319} display="flex" borderLeft={1} borderColor={silverSmokeColor}>
+      <Box flexGrow={1} width={319} display="flex" borderColor={silverSmokeColor} className={classes.sideBarWrapper}>
```

Comment:
If you have a className, it's cleaner to move  borderColor also there

Reply:
remove `borderColor`

Reply:
Shouldn't this be correct for other style attributes like position, width, display?
Is there any disadvantage / advantage implementing styles using one way or the other?